conf/config_validator.go

```diff
@@ -0,0 +1,110 @@
+package conf
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/aerospike/aerospike-management-lib/asconfig"
+	"github.com/go-logr/logr"
+)
+
+var (
+	ErrConfigValidation = fmt.Errorf("error while validating config")
+)
+
+type ConfigValidator struct {
+	ConfHandler
+	mgmtLogger logr.Logger
+	version    string
+}
+
+func NewConfigValidator(confHandler ConfHandler, mgmtLogger logr.Logger, version string) *ConfigValidator {
+	return &ConfigValidator{
+		ConfHandler: confHandler,
+		mgmtLogger:  mgmtLogger,
+		version:     version,
+	}
+}
+
+// Validate validates the parsed configuration in ac against
+// the Aerospike schema matching ac.aerospikeVersion.
+// ValidationErrors is not nil if any errors during validation occur.
+// ValidationErrors Error() method outputs a human readable string of validation error details.
+// error is not nil if validation, or any other type of error occurs.
```

Comment:
This comment needs to be updated to match the current state of the file.
For example, there is no more `ac.aerospikeVersion`.
The lines about `ValidationError` Error() method should be moved to that method.